makefile

```diff
@@ -1,5 +1,30 @@
-all: modern
+OBJS= compress.o aws2c.o
+CFLAGS= -O2
 
-modern: aws2c.c compress.h compress.c
-	gcc aws2c.c compress.c -o aws2c
+all: aws2c
 
+aws2c: $(OBJS)
+	$(CC) -o $@ $(OBJS) $(LDFLAGS)
+
+demox: commanderX16.o
+	cl65 -t cx16 -o $@ -D DEMOX $?
+
+loadsave.o: loadsave.c config.h
```

Comment:
IIRC, one does not need loadsave to be compiled with gcc, but only to be available so that the cross compiler can compile them for the target machine.